src/cpp/utils/IPFinder.cpp

```diff
@@ -83,7 +83,6 @@ bool IPFinder::getIPs(std::vector<info_IP>* vec_name, bool return_loopback)
 
                 if (family == AF_INET || family == AF_INET6) //IP4
                 {
-                    //printf("\t%s ",  family == AF_INET ? "IPv4":"IPv6");
```

Comment:
Why should the patch remove this comment?

Reply:
This would produce undesired console output, so I don't mind it being removed.